addon-test-support/@ember/test-helpers/dom/wait-for.js

```diff
@@ -15,7 +15,7 @@ import { nextTickPromise } from '../-utils';
   @param {number} [options.count=null] the number of elements that should match the provided selector (null means one or more)
   @returns {Element|Array<Element>} the element (or array of elements) that were being waited upon
 */
-export default function waitFor(selector, { timeout = 1000, count = null } = {}) {
+export default function waitFor(selector, { timeout = 1000, count = null, timeoutMessage = 'waitFor timed out' } = {}) {
```

Comment:
I can revert the default and keep `waitUntil timed out` if you would like.

Reply:
Nah, I think this is fine...